composer.json

```diff
@@ -9,7 +9,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.2.5",
-        "beyondcode/laravel-websockets": "^1.4",
```

Comment:
Not having dev-master seemed to be causing a 500 error on the dashboard.  This resolves that.